projects/1.20/assets/sanity-descent-into-madness/sanitydim/lang/zh_cn.json

```
@@ -0,0 +1,36 @@
+{
+	"entity.sanitydim.rotting_stalker": "腐烂跟踪者",
+	"entity.sanitydim.sneaking_terror": "惊骇潜行者",
+	
+	"commands.sanity.set.success.single": "将%s的理智设置为%s",
+	"commands.sanity.set.success.multiple": "将%2$s名玩家的理智设置为%1$s",
+	"commands.sanity.add.success.single": "为%2$s增加了%1$s理智",
+	"commands.sanity.add.success.multiple": "为%2$s名玩家增加了%1$s理智",
+	"commands.sanity.config.reload": "成功重载了维度配置文件",
+	
+	"gui.sanitydim.hint00": "这是什么动静？",
+	"gui.sanitydim.hint01": "我感觉有点不对",
+	"gui.sanitydim.hint02": "那是什么？",
+	"gui.sanitydim.hint03": "好奇怪的感觉...",
```

```suggestion
	"gui.sanitydim.hint03": "好奇怪的感觉……",
```
可以整个文件都改一下，中文输入法下Shift+6